spec/serverSpec.js

```diff
@@ -2,28 +2,52 @@
  * for these tests to pass. */
 
 const { expect } = require('chai');
-const request = require('request');
+const axios = require('axios');
 
 describe('the server', () => {
-  it('the data/documents are in the right format & adhere to the schema', (done) => {
-    request('http://localhost:3003/reviews/20', (error, response, body) => {
-      const result = JSON.parse(body);
-      const review = result[0];
-
-      expect(review).to.have.all.keys('_id', 'id', 'image_url', 'reviewer_name', 'star_rate', 'review_date', 'review_description', 'likes_count');
-      done();
-    });
+  let result;
+
+  // runs before all tests in this block
+  before((done) => {
+    axios('http://localhost:3003/reviews/20')
+      .then((response) => {
+        result = response.data;
+        done();
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  });
+
+  it('the data/documents are in the right format & adhere to the schema', () => {
+    const review = result[0];
+
+    expect(review).to.have.all.keys('_id', 'id', 'image_url', 'reviewer_name', 'star_rate', 'review_date', 'review_description', 'likes_count');
+  });
+
+  it('the reviews are the ones associated to the book through its id', () => {
```

Comment:
Minor suggestion: adjust wording of the test to be a little more clear. For example: it('each displayed review is for the current book being viewed').